libs/template/src/lib/render-strategies/strategies/native.strategy.ts

```diff
@@ -3,17 +3,18 @@ import { tap } from 'rxjs/operators';
 import { ɵmarkDirty as markDirty } from '@angular/core';
 
 /**
- * Native Strategy
  * @description
  *
- * - mFC - `cdRef.markForCheck`
+ * Native Strategy
+ *
+ * @description
  *
  * This strategy mirrors Angular's built-in `async` pipe.
  * This means for every emitted value `ChangeDetectorRef#markForCheck` is called.
  *
- * | Name        | ZoneLess | Render Method | ScopedCoalescing | Scheduling | Chunked |
- * |-------------| ---------| --------------| ---------------- | ---------- |-------- |
- * | `native`    | ❌       | mFC           | ❌                | ❌         | ❌      |
+ * | Name      | Zone Agnostic | Render Method   | Coalescing    | Scheduling |
+ * | --------- | --------------| --------------- | ------------- | ---------- |
+ * | `native`  | ❌             | `markForCheck` | ❌             | ❌         |
```

Comment:
see comment above, scheduling and coalescing is built in

Reply:
Done